### PR TITLE
Add missing features from Jenkins MATLAB plugin

### PR DESCRIPTION
**Pull request by abhargav-mw**
This PR adds support for the following in matlab/run-tests command,
- Model Coverage in Cobertura format.
- Test report generation in PDF format.
- Simulink Test result file generation.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -121,6 +149,15 @@ jobs:
       - matlab/run-tests:
           test-results-junit: test-results/matlab/filterdtagresult.xml
           select-by-tag: FILTERED
+    # Set up for model coverage artifact tests
+      - matlab/run-command:
+          command: cd simtests;createModelAndTest;
+      - matlab/run-tests:
+          select-by-folder: simtests
+          model-coverage-cobertura: model-coverage/coverage.xml
+          test-results-pdf: test-resuts/matlab/pdfresults.pdf
```
Comment (mcafaro) [inline .circleci/config.yml:158]: `test-resuts` -> `test-results`

File: .circleci/config.yml [hunk 2]
```diff
@@ -108,7 +109,34 @@ jobs:
...
+                cs.MetricSettings = 'd';
+                ts = tf.getTestSuites;
+                tc = ts.getTestCases;
+                tc.setProperty('model', model);
+                tf.saveToFile;
+                tf.close;
+                sltest.testmanager.close;
+            end
+            disp('Created Model and Simulink Test file to simulate the model.');
+            _EOF
```
Comment (mcafaro) [inline .circleci/config.yml:137]: This setup logic is getting sophisticated enough where I think we should begin to consider pre-creating these files and checking them in.

I think this is OK for now though.

File: src/commands/run-tests.yml [hunk 3]
```diff
@@ -36,6 +36,21 @@ parameters:
...
     type: string
     default: ''
+  model-coverage-cobertura:
+    description: >
+      Path to write model coverage report in Cobertura XML format.
+    type: string
+    default: ''
+  test-results-simulink-test:
+    description: >
+      Path to write Simulink Test results in MLDATX format for Simulink Test test files.
```
Comment (mcafaro) [inline src/commands/run-tests.yml:46]: This is the language we use in Azure.
```suggestion
      Path to export Simulink Test Manager results in MLDATX format.
```

**Review by mcafaro — APPROVED** [review]
Overall looks good. Just some minor suggestions.

File: .circleci/config.yml (hunk 1 above)
Comment (abhargav-mw) [reply .circleci/config.yml]: Done.

File: .circleci/config.yml (hunk 2 above)
Comment (abhargav-mw) [reply .circleci/config.yml:137]: I agree, probably we could commit these files as part of next set of changes.

File: src/commands/run-tests.yml (hunk 3 above)
Comment (abhargav-mw) [reply src/commands/run-tests.yml]: Done.

**Pull request merged**